dev-python/PythonQt/files/PythonQt-3.1-commit-433-441.patch

```diff
@@ -0,0 +1,301 @@
+diff -uprNEBZ --suppress-blank-empty a/extensions/PythonQt_QtAll/PythonQt_QtAll.cpp b/extensions/PythonQt_QtAll/PythonQt_QtAll.cpp
```

Comment:
No, it does not.

> # repoman -d full
> RepoMan scours the neighborhood...
> RepoMan sez: "If everyone were like you, I'd be out of business!"

and I use "repoman commit".

I can try to decrease patch size, throwing out everything except QtUiTools fix. This will make it 7 times lesser.